src/index.json

```
@@ -3480,6 +3480,52 @@
                 "sha256Digest": "bbf330942daac0035bd9bb9a0fdeaf297124ec8cd98e28e46adb2bf34070295b"
             }
         ],
+        "baremetal-infrastructure": [
```

```suggestion
        "baremetal": [
```

I think the name should be the same as the name property value of `extensions`